packages/flags/package.json

```diff
@@ -27,20 +27,20 @@
       "require": "./dist/index.cjs"
     },
     "./next": {
-      "import": "./dist/next/index.js",
-      "require": "./dist/next/index.cjs"
+      "import": "./dist/next.js",
```

Comment:
I think I've ran into issues when doing this in the past, either for `exports` or `typesVersions`, especially when importing this package.

Did you test it with a snapshot release?

Reply:
I've been iterating on the examples so far, but once they're passing, I'm looking to test a snapshot on an example ensuring it has legacy module resolution (`moduleResolution: node`)